app/src/main/java/com/mapbox/vision/examples/activity/ar/ArNavigationActivity.kt

```diff
@@ -48,26 +83,71 @@ class ArNavigationActivity : AppCompatActivity() {
         back.setOnClickListener {
             onBackPressed()
         }
+
+        val builder = MapboxNavigationOptions
+                .builder()
+                .enableOffRouteDetection(true)
+        mapboxNavigation = MapboxNavigation(this, Mapbox.getAccessToken()!!, builder.build())
```

Comment:
Can store accessToken to variable.

Reply:
Yes we can but Mapbox should be initialised at that moment so I think it is one more checking for it

Reply:
I mean we can `val token = Mapbox.getAccessToken()!!` and then reuse the variable here and below